Bugfix: Fixed Example projects so they work+ added ui improvement to help users.

grantkemp opened the pull request:
bugfix: Removed Client Secret checking validation as it was impossible to run the app whilst leaving the client secret blank.  It would just re-open the validation sheet.

feature: added placeholder text to tell the user that they should leave the client secret blank. I left the fields in, just in case they are going to be used for other projects/ in the future.
admin: Added comments to state that the client secret should be blank so its understood easily by developers trying to use this.

Hope these improvements help -
Tested ok on latest Xcode 7.3.1

thomasvl commented on Examples/CalendarSample/CalendarSampleWindowController.m:
```diff
@@ -930,14 +930,15 @@ - (void)deleteSelectedACLRule {
-  // hardcoded into the source, but the sample application asks the
-  // developer for the strings
+    // Applications should have client ID hardcoded into the source.
+    //  but the sample application asks the developer for the strings.
```
Extra space before "bug"

thomasvl commented on Examples/CalendarSample/CalendarSampleWindowController.m:
```diff
@@ -930,14 +930,15 @@ - (void)deleteSelectedACLRule {
-  if (clientID.length == 0 || clientSecret.length == 0) {
-    // Remind the developer that client ID and client secret are needed
+  if (clientID.length == 0) {
+    // Remind the developer that client ID is needed.  Client Secret is now left blank // S
```
Stray "// S" on the end?

thomasvl commented on Examples/DriveSample/DriveSampleWindowController.m:
```diff
@@ -812,14 +812,14 @@ - (NSString *)MIMETypeFileName:(NSString *)path
-  // hardcoded into the source, but the sample application asks the
-  // developer for the strings.
+    // Applications should have client ID hardcoded into the source.
+    //  but the sample application asks the developer for the strings.
```
Same extra space.

grantkemp commented:
@thomasvl  Thanks for looking.  I hope the fix above has addressed the review points. I removed one extra full stop to make sure it reads a bit clearer.
